### PR TITLE
Do homework 3

### PR DESCRIPTION
**Pull request by tayvazyan**

### CODE REVIEW

File: manual/lesson_3/homework_3/UHC.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,520 @@
...
+							"    pm.expect(jsonData.customer.email).to.eql((pm.environment.get(\"randomEmail\")));",
+							"});",
+							"",
+							"pm.test(\"Test data type of the response\", () => {",
+							"  pm.expect(jsonData).to.be.an(\"object\");",
+							"  pm.expect(jsonData.customer.email).to.be.a(\"string\");",
+							"  pm.expect(jsonData.customer.firstname).to.be.a(\"string\");",
+							"  pm.expect(jsonData.customer.lastname).to.be.a(\"string\");",
+							"  pm.expect(jsonData.customer.group.title).to.be.a(\"string\");",
+							"  pm.expect(jsonData.customer.uvp.planId).to.be.a(\"integer\");",
```
Comment (valeriyazin) [inline manual/lesson_3/homework_3/UHC.postman_collection.json]: please change 'integer' to 'number'

File: manual/lesson_3/homework_3/UHC.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,520 @@
...
+							"  pm.expect(jsonData.customer.uvp.planId.title).to.be.a(\"string\");",
```
Comment (valeriyazin) [inline manual/lesson_3/homework_3/UHC.postman_collection.json]: this field doesn't exist in the  documentation

**Review by valeriyazin — CHANGES_REQUESTED** [review]

File: manual/lesson_3/homework_3/UHC.postman_collection.json (hunk 1 above)
Comment (tayvazyan) [reply manual/lesson_3/homework_3/UHC.postman_collection.json]: Thanks, done

File: manual/lesson_3/homework_3/UHC.postman_collection.json (hunk 2 above)
Comment (tayvazyan) [reply manual/lesson_3/homework_3/UHC.postman_collection.json]: Thanks, fixed

**Review by valeriyazin — APPROVED** [review]

**Pull request merged**